### PR TITLE
graphene: update to 1.10.0

### PR DESCRIPTION
**Pull request by streaksu**

### CODE REVIEW

**Comment by Duncaen** [comment]
One commit per template.

**Comment by streaksu** [comment]
@Duncaen It was a revision bump tho, I thought we were supposed to add those inside the updated package that causes the bump.

**Comment by Duncaen** [comment]
Never with the same commit, but in one pull request.

File: common/shlibs [hunk 1]
```diff
@@ -3624,7 +3624,7 @@ libFAudio.so.0 FAudio-19.05_1
 libqaccessibilityclient-qt5.so.0 libqaccessibilityclient-0.4.0_1
 libnitrokey.so.3 libnitrokey-3.4.1_1
 libceres.so.1 ceres-solver-1.14.0_1
-libgraphene-1.0.so.0 graphene-1.8.2_1
+libgraphene-1.0.so.0 graphene-1.10.0_1
```
Comment (xtraeme) [inline common/shlibs]: is it really necessary to bump min version?

if true, why?

File: srcpkgs/graphene/template [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 pkgname=graphene
-version=1.8.2
-revision=2
+version=1.10.0
+revision=1
 build_style=meson
 build_helper="gir"
 configure_args="-Dtests=false -Dbenchmarks=false
- -Dintrospection=$(vopt_if gir true false)"
+ -Dintrospection=$(vopt_if gir true false) -Darm_neon=false"
```
Comment (xtraeme) [inline srcpkgs/graphene/template]: I suspect you've disabled neon simply to unbreak the build on armv6, isn't it?

**Comment by streaksu** [comment]
@Duncaen Fixed, I hope thats okay

File: common/shlibs (hunk 1 above)
Comment (streaksu) [reply common/shlibs]: I thought we were supposed to make sobumps each time we update libraries, sorry if thats not the case.

Comment (xtraeme) [reply common/shlibs]: nope unless:

- there's a SONAME bump
- a dependency needs symbols available in a recent version

File: srcpkgs/graphene/template (hunk 2 above)
Comment (streaksu) [reply srcpkgs/graphene/template]: If I remember correctly NEON is not architectural in armv6 and armv7, so apart of circumventing the error we can ensure compatibility with all armv* devices regardless of the extensions present on their CPUs.

File: common/shlibs (hunk 1 above)
Comment (streaksu) [reply common/shlibs]: Well if thats the case I was working on a `mutter` update that needs the new symbols provided (at least `meson` enforces such version), but I didn't know that back then.

File: srcpkgs/graphene/template (hunk 2 above)
Comment (jnbr) [reply srcpkgs/graphene/template]: Neon was introduced with armv7 but support was optional. Void does not use neon for armv7 (with some exceptions).  So for armv6 and armv7 this is ok, but it should not be disabled for armv8.

File: common/shlibs (hunk 1 above)
Comment (abenson) [reply common/shlibs:3627]: Package version shouldn't need to be changed in common/shlibs unless the soname/symbols change.

**Comment by sgn** [comment]
1.10.0 had been merged.

**Pull request closed without merging**